src/taxi/utils.py

```diff
@@ -0,0 +1,127 @@
+import os
+from enum import Enum
+from django.conf import settings
+from odata_request_parser.main import OdataFilterParser, OdataSelectParser
+
+from main.utils import DecosBase, ImmediateHttpResponse
+
+
+class DecosBusiness(Enum):
+    taxi = "TAXXXI"
+    folders = "FOLDERS"
+
+
+class DecosTaxi(DecosBase):
+    ZAAKNUMMER = {
+        "BSN": settings.TAXI_BSN_ZAAKNUMMER,
+        "ZONE_ONTHEFFING": settings.TAXI_ZONE_ONTHEFFING_ZAAKNUMMER,  # Permits
+        "HANDHAVINGSZAKEN": settings.TAXI_HANDHAVINGSZAKEN_ZAAKNUMMER,  # Enforcement cases
+    }
+
+    def get_taxi_zone_ontheffing(self, driver_bsn: str) -> list[dict]:
+        """
+        request the permit from a driver based on their bsn nr
+        """
+        decos_key = self.get_driver_decos_key(driver_bsn)
+        driver_permits = self.get_driver_ontheffing_en_handhaving(decos_key)
+        return [{'vergunningsnummer': p} for p in driver_permits]
+
+    def get_driver_decos_key(self, driver_bsn: str) -> str:
+        """
+        Request the decos key from decosdvl for a driver based on their bsn nr
+        """
+
+        class DecosParams(Enum):
+            bsn = "NUM1"
+            a_nummer = "TEXT13"
```

Comment:
Waar dient dit a_nummer voor? Ik zie het nergens anders gebruikt worden..

Reply:
Dit is puur zodat we voor de toekomst weten dat dit veld het a-nummer is. Mogelijk hebben we dat nog eens nodig. Zie het als documentatie